### PR TITLE
MGMT-3839 - update test_infra to support new static_ips API format

### PR DESCRIPTION
**Pull request by yevgeny-shnaidman**

### CODE REVIEW

File: discovery-infra/test_infra/tools/static_ips.py [hunk 1]
```diff
@@ -78,8 +70,8 @@ def _genetate_ips(mac_addresses, cidrs, address_offset):
         ipv6_conf = next(ipv6_gen)
         static_ips.append({
             'mac': mac,
-            'ip': ipv4_conf.ip, 'mask': ipv4_conf.mask, 'gateway': ipv4_conf.gateway, 'dns': ipv4_conf.dns,
-            'ip_v6': ipv6_conf.ip, 'mask_v6': ipv6_conf.mask, 'gateway_v6': ipv6_conf.gateway, 'dns_v6': ipv6_conf.dns
+            'ipv4_config': ipv4_conf,
+            'ipv6_config': ipv6_conf
```
Comment (empovit) [inline discovery-infra/test_infra/tools/static_ips.py:74]: You don't need the ipv4_conf/ipv6_conf variables any more.
```
'ipv4_config': next(ipv4_gen)
'ipv6_config': next(ipv6_gen)
```

Comment (empovit) [reply discovery-infra/test_infra/tools/static_ips.py:74]: Are the service changes already in? AFAIK it's a breaking API change.

Comment (yevgeny-shnaidman) [reply discovery-infra/test_infra/tools/static_ips.py]: fixed

Comment (yevgeny-shnaidman) [reply discovery-infra/test_infra/tools/static_ips.py]: and the service changes are in

**Pull request merged**